doc/man3/OCSP_sendreq_new.pod

```diff
@@ -35,14 +35,14 @@ response header maximum line length of B<maxline>. If B<maxline> is zero a
 default value of 4k is used. The OCSP request B<req> may be set to B<NULL>
 and provided later if required.
 
-OCSP_sendreq_nbio() performs non-blocking I/O on the OCSP request context
-B<rctx>. When the operation is complete it returns the response in B<*presp>.
+OCSP_sendreq_nbio() performs I/O on the OCSP request context B<rctx>.
```

Comment:
Why removal of "non-blocking"?

Reply:
I've removed "non-blocking" in the documentation in order to rectify an inconsistency: 
already quite some time ago the `_nbio` and `_bio` functions where unified, and whoever did this forgot to update the documentation accordingly. 
The `_nbio` prevailed likely for backward compatibility.